microsoft/testsuites/core/vm_resize.py

```diff
@@ -143,6 +158,10 @@ def _verify_vm_resize(
         assert expected_vm_capability, "fail to find proper vm size"
         if not hot_resize:
             start_stop.start()
+
+        runbook = node.capability.get_extended_runbook(AzureNodeSchema)
+        test_result.information["final_vm_size"] = runbook.vm_size
```

Comment:
Make the resize method return resized new vm size, instead of read it form runbook. So the test case doesn't need to depend on Azure specific runbook.

Reply:
any other way to get origin vm size besides get from runbook?
```
        node = cast(RemoteNode, environment.nodes[0])
        runbook = node.capability.get_extended_runbook(AzureNodeSchema)
        test_result.information["origin_vm_size"] = runbook.vm_size
```

Reply:
How about return a tuple with two vm sizes from `resize`, (origin, new)?

Reply:
updated